test/bin/manage_brew_rpms.sh

```diff
@@ -6,7 +6,7 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # its execution in a containerized environment with limited set of tools.
 
 usage() {
-    echo "Usage: $(basename "$0") [access | download <version> <path>]"
+    echo "Usage: $(basename "$0") [access | download <version> <path> <version_type>]"
```

Comment:
We also need to add a message about supported `version_type` values.

Reply:
done in https://github.com/openshift/microshift/pull/4900/commits/45975296e2817f9b8f7be0b23f0202fad1ab29e1 commit